src/components/CardOverlay.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {View} from 'react-native';
+import themeColors from '../styles/themes/default';
+
+const CardOverlay = () => (
+    <View
+        style={{
+            backgroundColor: themeColors.modalBackdrop,
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            opacity: 0.5,
```

Comment:
This style should be moved into `styles.js`. We have standardized on avoiding inline styles and there is some more explanation [here](https://github.com/Expensify/Expensify.cash/blob/main/STYLING.md#inline-styles).

Reply:
Done!